staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -116,7 +116,8 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringSliceVar(&s.EtcdServersOverrides, "etcd-servers-overrides", s.EtcdServersOverrides, ""+
 		"Per-resource etcd servers overrides, comma separated. The individual override "+
-		"format: group/resource#servers, where servers are URLs, semicolon separated.")
+		"format: group/resource#servers, where servers are URLs, semicolon separated. "+
+		"Note that etcd servers overrides currently do not support CRDs.")
```

Comment:
The k8s.io/apiserver component is not aware of CRDs... not sure this help text belongs here.

Reply:
I rephrased the comment as suggested, please check again.